### DISCUSSION TITLE
Can Homebrew be added to docker-osx:auto ?

### DISCUSSION THREAD

**Discussion by ggjulio** [sickcodes/Docker-OSX#217] [Ideas, Suggestions, Requests]
[Homebrew](https://brew.sh/) is a very handy package manager. But it takes so long to install it.
It would be great if it was included to the build.

Or maybe create a new tag with all very common tools added to the build ?

For example, here is the [packages list installed by default into github's macos image](https://github.com/actions/virtual-environments/blob/main/images/macos/macos-10.15-Readme.md)

**Comment by sickcodes** [comment]
Yeah looking into it, it's quite large though, 11GB zipped. Not sure if it will fit yet in the hub 🤣

**Reply by sickcodes** [reply]
```bash
# boot to OS X shell + display + specify commands to run inside OS X!
docker run -it \
    --name harry \
    --device /dev/kvm \
    -p 50922:10022 \
    -v /tmp/.X11-unix:/tmp/.X11-unix \
    -e "DISPLAY=${DISPLAY:-:0.0}" \
    -e "OSX_COMMANDS=/bin/bash -c \"\$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)\"" \
    sickcodes/docker-osx:auto
```

Then after it's finished overnight, start it up with `docker start -ai harry`
